source/features/clean-checks-list.css

```diff
@@ -6,7 +6,7 @@
 /* .text-italic targets "running" or "waiting" checks */
 
 /* Required to allow sorting */
-.merge-status-list {
+.merge-status-list.js-updatable-content-preserve-scroll-position {
```

Comment:
It's best to leave a comment to clarify why this is needed (e.g. `GitHub misuses the merge-status-list class on review lists in the mergeability box`)